services/src/main/java/org/keycloak/protocol/oidc/OIDCWellKnownProvider.java

```diff
@@ -140,11 +142,7 @@ public void close() {
     }
 
     private static List<String> list(String... values) {
-        List<String> s = new LinkedList<>();
-        for (String v : values) {
-            s.add(v);
-        }
-        return s;
+        return new LinkedList<>(Arrays.asList(values));
```

Comment:
Can't this be further improved to just Arrays.asList(values)?

Reply:
Original code returned LinkedList. I have tried to save semantic. Of cource I can change it, but I'm not sure about the meaning.  As you wish :)

Reply:
the method returns a List, so the expectation is that it returns a list and doesn't have to be a LinkedList.

Reply:
Done